original_sdk/tests/test_async_client_e2e.py

```diff
@@ -127,10 +127,12 @@ async def test_edit_asset(self, async_client: OriginalAsyncClient):
         while is_transferable is False and retries < TEST_RETRY_COUNTER:
             response = await async_client.get_asset(asset_uid)
             is_transferable = response["data"]["is_transferable"]
-            time.sleep(15)
+            await asyncio.sleep(15)
```

Comment:
Note: this seems to be the way to sleep when dealing with async (fixed flakey tests)